oauth2client/service_account.py

```diff
@@ -107,9 +126,6 @@ def serialization_data(self):
         'private_key': self._private_key_pkcs8_text
     }
 
-  def create_scoped_required(self):
```

Comment:
This should be left in for compatibility, although it can be changed to return false.